src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -890,13 +901,13 @@ def getStatsFromMongoDB(self, detail=False, **kwargs):
                     mongoFilter = {'name': rseName}
                     data["rseData"].append(self.msUnmergedColl.find_one(mongoFilter, projection=mongoProjection))
 
-                # Rewrite all timestamps in ISO 8601 format
-                for rse in data['rseData']:
-                    if 'timestamps' in rse:
-                        for dateField, dateValue in rse['timestamps'].items():
-                            dateValue = datetime.utcfromtimestamp(dateValue)
-                            dateValue = dateValue.isoformat()
-                            rse['timestamps'][dateField] = dateValue
+            # Rewrite all timestamps in ISO 8601 format
+            for rse in data['rseData']:
+                if 'timestamps' in rse:
+                    for dateField, dateValue in rse['timestamps'].items():
+                        dateValue = datetime.utcfromtimestamp(dateValue)
```

Comment:
I suggest to add extra validation check to see if `dateValue` is indeed timestamp (second since epoc), otherwise you may pass anything here and code will break

Reply:
It is validated when the entry is recorded in the database. And in fact it is not a timestamp, but a regular integer - seconds from start of epoch. If we get anything else here, it would mean the whole service is broken in many many other and more important places. This code block here is just for translating the epoch seconds into readable data format and to display them in the following rest interface: 

https://cmsweb.cern.ch/ms-unmerged/data/info?rse=T2_US_Florida 

This is already tested and is working in production. The problem was that I have made a minor mistake to put it under the `if else` statement and it was working for the REST interface responsible for a single RSE, while for the whole set of RSEs it was still visualizing the field as an integer:

https://cmsweb.cern.ch/ms-unmerged/data/info?rse=ALL_DOCS

This REST interface plays just an informative role. In no way we can break the service from this method.